src/index.js

```diff
@@ -38,7 +38,7 @@ class Watch {
     this.rightNow = Moment();
 
     settings.dials.forEach((dial) => {
-      let tempDial = new Dial(dial, this);
+      let tempDial = new Dial(dial, this, parentId);
```

Comment:
I feel the ID should exist as a property on the parent Watch class. So when passing `this` to any of the components, the ID is accessible that way.